descriptors/ModuleDescriptor-template.json

```diff
@@ -462,6 +462,17 @@
         }
       ]
     },
+    {
+      "id": "finance-storage.finance-data",
```

Comment:
we also need to keep in mind this story for Bama team, but here data should be returned in hierarhical format
https://folio-org.atlassian.net/browse/MODFISTO-506
so most of all they need to implement new DB View, do you agree?

Reply:
agree